policy/modules/system/authlogin.if

```diff
@@ -719,6 +719,8 @@ interface(`auth_manage_shadow',`
 	auth_manage_shadow_history($1)
 	auth_rw_shadow_lock($1)
 	allow $1 shadow_t:file manage_file_perms;
+	auth_etc_filetrans_shadow($1, "shadow-")
+	auth_etc_filetrans_shadow($1, "gshadow-")
```

Comment:
Please update the existing `auth_etc_filetrans_shadow` instead.

Reply:
I handled this slightly differently, but hopefully it addresses your concern.